jerry-core/jerry.c

```diff
@@ -1848,7 +1848,8 @@ snapshot_add_compiled_code (ecma_compiled_code_t *compiled_code_p, /**< compiled
 
   snapshot_last_compiled_code_offset = snapshot_buffer_write_offset;
 
-  compiled_code_map_entry_t *new_entry = (compiled_code_map_entry_t *) mem_pools_alloc ();
+  compiled_code_map_entry_t *new_entry =
+  (compiled_code_map_entry_t *) mem_heap_alloc_block (sizeof (compiled_code_map_entry_t));
```

Comment:
No. Seriously. I - will - not - add - extra - statements, - temporaries, - assignments - just - because - of - a - style - checker.

Style checkers should promote readability. Nicer style. But as soon as they force you to write _different_ (not differently styled!) code than you think it's optimal, then there is something terribly wrong.

(And the other issue is: the temp should be of type `compiled_code_map_entry_t *`, I guess. But then, there is no way to get that line under the 120 chars limit. Even a one-character temp would make a 121 chars line. So, that line should be broken somewhere too, which causes the same issue as we have now.)